### PR TITLE
Feat/hng 45 about us page

### PR DESCRIPTION
**Pull request by MUdugbesi**
# Description

**Closes #issue_number_here** #45

# Changes proposed

### I was told to Implement both the mobile and web view of the about us page . Also to implement every content in the page excluding the header and the footer, use already made header and footer if there is a need to.

Page heading section with all the graphic assets

- Stats section
- Our service section
- Our mission section
- Our vision section
- Our core values section

- About Us page

## What did you do?
- Implement all the sections in the design
- Made page responsive across (mobile and web) screens
- I made use of reusable components where needed
- I Wrote test for the implementations
- Used proper commit message

- Added a new folder for images in public folder - titled about us
- Added a new Folder for the about us page in Layouts
- No dependencies installed
- Added a folder for about page test in the test folder

# Check List (Check all the applicable boxes)
Lint test
UI test
Screenshot web - 1
Screenshot web - 2
Screenshot web 3
Screenshot web - 4
Simulator Screenshot - 1 png
Simulator Screenshot - 2
Simulator Screenshot - 3 png
Simulator Screenshot - 4
Simulator Screenshot - 5
Simulator Screenshot -
Simulator Screenshot - 6
Simulator Screenshot - 7
Simulator Screenshot - 8

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/Videos

### CODE REVIEW

**Comment by MUdugbesi** [comment]
Hi Mentors, could you please review this PR. Thanks

File: src/components/layouts/AboutUs/AboutUs.tsx
Comment (Samadeen) [inline src/components/layouts/AboutUs/AboutUs.tsx:1]: Why is your page in the component folder?

File: src/components/layouts/AboutUs/Blog.tsx
Comment (Samadeen) [inline src/components/layouts/AboutUs/Blog.tsx:1]: I'm curious to know if you know what layout means and why you have your components in them

**Review by Samadeen — CHANGES_REQUESTED** [review]
You might have to restructure your folder structure

File: src/components/layouts/AboutUs/AboutUs.tsx
Comment (MUdugbesi) [reply src/components/layouts/AboutUs/AboutUs.tsx:1]: I can see the homepage in the layouts -

File: src/components/layouts/AboutUs/Blog.tsx
Comment (MUdugbesi) [reply src/components/layouts/AboutUs/Blog.tsx:1]: The arrangement of the elements on a Page?? I am trying not to mess up the structure, hence didn't create another folder - do I create a new folder for my static page or could you advise where my page should be.

Comment (MUdugbesi) [reply src/components/layouts/AboutUs/Blog.tsx:1]: Or we could have a pages folder too??

**Pull request closed without merging**